bin/travis/tools.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
there's no need for separate file (for now ;) ). It can be a function in `test.sh` or even a direct variable in `install_bloop` function:
```sh
bloopVersion=$(sed -n 's|val\s*bloop\s*=\s*"\(.*\)"\s*$|\1|p' build.sbt)
```